cEpiabm/test/dataclasses/test_place.cpp

```diff
@@ -1,25 +1,149 @@
 
+#include "dataclasses/population.hpp"
 #include "dataclasses/place.hpp"
 
 #include "../catch/catch.hpp"
 
 #include <random>
+#include <iostream>
 
 using namespace epiabm;
+
+inline Population makeSubjectPlaceTest(size_t n_places, size_t n_cells, size_t n_people)
+{
+    Population subject = Population();
+    subject.cells().reserve(n_cells);
+    subject.places().reserve(n_places);
+    for (size_t i = 0; i < n_places; i++)
+    {
+        subject.places().push_back(Place(i));
+    }
+    for (size_t i = 0; i < n_cells; i++)
+    {
+        subject.cells().push_back(Cell(i));
+        subject.cells()[i].people().reserve(n_people);
+        for (size_t j = 0; j < n_people; j++)
+        {
+            subject.cells()[i].people().push_back(Person(i, j, j));
+
+            /*printf("Microcell: %ld, Person: (%ld, %ld)\n",
+                i,
+                subject.people()[subject.microcells()[i].people()[j]].cellPos(),
+                subject.people()[subject.microcells()[i].people()[j]].microcellPos());//*/
```

Comment:
```suggestion
```

Fine for testing but I think commented out code is generally not a good thing?

Reply:
Fine to have in the tests, this bit of code is pretty tricky, and its possible we'd have to modify it in the future so it'll be useful to have these prints handy. Very easy to mess up initialization and get segmentation faults with this function.

Reply:
Sorry I didn't notice this was a test file - all good!